test/edu-benefits/components/BenefitsSelectionFields.unit.spec.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import BenefitsSelectionFields from '../../../src/js/edu-benefits/components/BenefitsSelectionFields';
+
+describe('<BenefitsSelectionFields>', () => {
+  const data = {
+    benefitsRelinquished: {
+      value: ['chapter30'],
+      dirty: false
+    },
+    chapter33: true
+  };
+  const onStateChange = sinon.spy();
+
+  const tree = SkinDeep.shallowRender(
+    <BenefitsSelectionFields
+        data={data}
+        onStateChange={onStateChange}/>
+  );
+
+  it('should render a subsection for chapter 33', () => {
+    expect(tree.everySubTree('ErrorableRadioButtons').length).to.equal(1);
+  });
+  it('should render four checkboxes', () => {
```

Comment:
Yeah, I wasn't sure what the best approach was. I think this is probably too granular.